webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/domain/queries/CmdbQuery.scala

```diff
@@ -172,13 +172,15 @@ sealed trait CriterionType extends ComparatorList {
 // a case class that allows to precompute some parts of the NodeInfo matcher which are indep from the
 // the node.
 trait NodeInfoMatcher {
+  def debugString: String
```

Comment:
This one is added to be able to debug the part with node properties filtering: 
```
12:49:26.662 [main]            DEBUG com.normation.rudder.services.queries.AcceptedNodesLDAPQueryProcessor - LDAP result: 0 entries obtained in 291ms for query { returnType:'NodeReturnType' with 'Or' criteria [serializedNodeProperty.name.value eq foo=bar] }
12:49:27.008 [zio-default-blocking-1-2117173674] TRACE com.normation.rudder.services.queries - node6: NOK for [Prop name=value equals'foo=bar']
12:49:27.008 [zio-default-blocking-1-2117173674] TRACE com.normation.rudder.services.queries - node7: NOK for [Prop name=value equals'foo=bar']
12:49:27.009 [zio-default-blocking-1-2117173674] TRACE com.normation.rudder.services.queries - node3: NOK for [Prop name=value equals'foo=bar']
12:49:27.009 [zio-default-blocking-1-2117173674] TRACE com.normation.rudder.services.queries - node0: NOK for [Prop name=value equals'foo=bar']
12:49:27.009 [zio-default-blocking-1-2117173674] TRACE com.normation.rudder.services.queries - node4: NOK for [Prop name=value equals'foo=bar']
12:49:27.009 [zio-default-blocking-1-2117173674] TRACE com.normation.rudder.services.queries - node2: NOK for [Prop name=value equals'foo=bar']
12:49:27.009 [zio-default-blocking-1-2117173674] TRACE com.normation.rudder.services.queries - node5: NOK for [Prop name=value equals'foo=bar']
12:49:27.010 [zio-default-blocking-1-2117173674] TRACE com.normation.rudder.services.queries - node1: OK for [Prop name=value equals'foo=bar']
12:49:27.010 [zio-default-blocking-1-2117173674] TRACE com.normation.rudder.services.queries - root: NOK for [Prop name=value equals'foo=bar']
12:49:27.027 [main]            DEBUG com.normation.rudder.services.queries.AcceptedNodesLDAPQueryProcessor - [post-filter:rudderNode] Found 1 nodes when filtering for info service existence and properties (366 ms)````